src/app/task/task.component.html

```diff
@@ -1,25 +1,28 @@
 <h2>Task To Do: </h2>
 
 <div>
-
     <form [formGroup]="taskForm" (ngSubmit)="addTask()">
-        <label> Name:
-      <input type="text" formControlName="name">
-    </label>
+        <label for="form1" class=''> Name:
+            <input mdbInput type="text" id="form1" class="form-control" formControlName="name">
+        </label>
         <label> Description:
-      <input type="text" formControlName="description">
-    </label>
+            <input mdbInput type="text" id="form1" class="form-control" formControlName="description">
+        </label>
+
 
-        <button type="submit">Add Task</button>
+        <button type="submit" mdbBtn mdbWavesEffect color="primary" title="Adding a task">
+            <p class="add-task">Add Task</p>
```

Comment:
You don't have to add button text in p tags. If you want to style it, wrap it in span. p is meant for paragraphs. span is for this kind of custom work

Reply:
https://stackoverflow.com/questions/1908234/when-to-use-span-instead-p